site/docs/guidelines/typography.mdx

```diff
@@ -32,71 +32,56 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 
 ### Headings
 
-- **Display 0:** Used for brand moments including, introductory screens (Onboarding, Success pages etc.) or landing pages (Log In and Capture Intro etc.).
+- **Display 0:** Used *only* for brand moments including introductory screens (Log In, Onboarding, Capture Intro etc) or outro pages (Positive, Negative etc.).
 
-- **Heading 1:** Used for page titles in the [Title Block](/guidelines/header/#title-block) of the page [Header](/guidelines/header/)
+- **Heading 1:** Used only for page titles in the [Title Block](/guidelines/header/#title-block) of the page [Header](/guidelines/header/) or Workflow Header.
 
-- **Heading 2:** Used for headings that introduce a major section of content. Generally outside of a card. Generally at the top of a content area, below the Title Block.
+- **Heading 2:** Also referred to as a Section Heading. Used to introduce major sections of content on a page, and in containers independent of the page hierarchy, such as [Modals](/components/modal/) or Drawers. Generally positioned outside of cards in most layouts, it can also be used inside if more appropriate to the surrounding context.
 
-- **Heading 3:** Used within a card as a heading.
+- **Heading 3:** Use to label subsets or groupings of information within a section, or as a title in individual objects, such as a [Tile](/components/tile/) or [Guidance Block](/components/guidance-block/). Generally used within a card container. 
 
-- **Heading 4:** Used to define the name/title of a survey, report or other primary information within a card.
+- **Heading 4:** Used for the primary piece of information in a list item, such as the title of a survey/report or an employee name.
 
-- **Heading 5:** Used in long form content when you have 5 heading levels. Consider breaking this level of detail into its own page with larger headings.
+- **Heading 5:** Primarily for long form content where an extra level of hierarchy is needed (such as Skills Coach, and educational content). Stylistically identical to Body Bold. **Avoid using Heading 5 with short form content in product screens.**
 
-- **Heading 6:** Used for table headers, or small headings above content, such as form inputs or defining categories of content.
+- **Heading 6:** Used for small headings on interactive elements, such as table headers, form inputs labels and in popovers and notifications.
 
```

Comment:
Could link to Tables (/components/table/), Popovers (/components/popover/), and Notifications (/guidelines/notifications/) here 

Think it should also be "form input labels"